packages/web-components/src/components/masthead/left-nav-menu-section.ts

```diff
@@ -222,16 +226,20 @@ class C4DLeftNavMenuSection extends HostListenerMixin(FocusMixin(LitElement)) {
       showBackBtn,
     } = this;
     return html`
-      <ul>
+      <ul part="masthead-side-nav-submemu-container">
         ${showBackBtn
           ? html`
               <li
+                part="masthead-side-nav-submemu"
```

Comment:
I see that you're referencing a CSS class to come up with the name here, but in this case, I think there are a couple issues with that:
1. We don't need to copy over typos
2. This isn't really functioning as a submenu. Rather, it's functioning as a menu item and a back button wrapper:
```suggestion
                part="menu-item back-button-wrapper"
```